builtin/credential/aws/path_config_identity.go

```diff
@@ -5,10 +5,34 @@ import (
 	"fmt"
 
 	"github.com/hashicorp/vault/sdk/framework"
+	"github.com/hashicorp/vault/sdk/helper/aliasmetadata"
 	"github.com/hashicorp/vault/sdk/helper/strutil"
 	"github.com/hashicorp/vault/sdk/logical"
 )
 
+// aliasMetadataFields is a list of the default alias metadata
+// added to tokens during login. The default alias type used
+// by this back-end is the role ID. Subsequently, the default
+// fields included are expected to have a low rate of change
+// when the role ID is in use.
+var aliasMetadataFields = &aliasmetadata.Fields{
+	Default: []string{
+		"account_id",
+		"auth_type",
+	},
+	AvailableToAdd: []string{
```

Comment:
This list is different per auth method, right?

Reply:
Well, since we decided to have the configuration parameter live at the `config/identity` level, it's back-end-wide, rather than specific to a role type. But for both types, "iam" and "ec2", the default alias is a "role_id" so we'd want to have identical defaults for alias metadata - fields that are stable at a role level.

Reply:
I think this highlights a small gap in our design. Since `config/identity` supports both `iam_alias` and `ec2_alias`, I'm thinking that there should probably be both `iam_metadata` and `ec2_metadata`. The same situation exists in GCP too. @briankassouf ?

Reply:
I know we're trying to move this PR forward quickly, so I have incorporated that feedback in this commit: https://github.com/hashicorp/vault/pull/8783/commits/3c2ed56cc59d832cde197f1a0ad21dd18545ad38. This will have broken tests, which I'm presently fixing, and will push a separate test-fixing PR. Just getting this code up now so it's available for review.

Reply:
Just pushed test fixes.